### PR TITLE
Make map loading modal facts shuffle again

### PR DESCRIPTION
**Pull request by qnlee**
This fixes the facts not shuffling (verified locally).
Still have to address the other stuff mentioned in issue description:
- double check that the shuffling is random
- determine if it makes sense to create setting to define setInterval time
---
Fixes [#1559](https://github.com/hackforla/311-data/issues/1559)
  - [ ] Up to date with `dev` branch
  - [x] Branch name follows [guidelines](https://github.com/hackforla/311-data/blob/master/GETTING_STARTED.md#feature-branching)
  - [ ] All PR Status checks are successful
  - [ ] Peer reviewed and approved

Any questions? See the [getting started guide](https://github.com/hackforla/311-data/blob/master/GETTING_STARTED.md)

### CODE REVIEW

File: components/FactModal/index.jsx [hunk 1]
```diff
@@ -26,9 +26,12 @@ export default function FactModal({ isLoading }) {
 
   useEffect(() => {
     let intervalId = null;
+    let factIndex = 0;
+
     if (isLoading) {
       intervalId = setInterval(() => {
-        setCurrentFactIndex(prev => (prev === facts.length - 1 ? 0 : prev + 1));
```
Comment (edwinjue) [inline components/FactModal/index.jsx:31]: This is much more readable and I love the refactor Anna! However, this makes me think perhaps we could get by with reusing the `currentFactIndex` state that is already defined on line 25. What do you think about something like this? 

```
intervalId = setInterval(() => {
      setCurrentFactIndex((currentFactIndex + 1) % facts.length);
}, seconds(5));
```

**Review by edwinjue — CHANGES_REQUESTED** [review]

**Comment by edwinjue** [comment]
@qnlee Great work, let me know what you think about my comments. If you are ok with them and can make those fixes, we can go ahead and approve the changes and have them merged.

I will create a separate issue for creating a setting in [settings](https://github.com/hackforla/311-data/blob/1136d2468e33596af11b3dd2292980317c6a78d7/settings/index.js) to define [setInterval time](https://github.com/hackforla/311-data/blob/1136d2468e33596af11b3dd2292980317c6a78d7/components/FactModal/index.jsx#L32)

File: components/Map/index.js [hunk 2]
```diff
@@ -397,7 +397,7 @@ class MapContainer extends React.Component {
         <CookieNotice />
         {(isDbLoading || isMapLoading) && (
           <>
-            <FactModal isLoading={isMapLoading} />
+            <FactModal isLoading={true} />
```
[the remark this answers is not in the document]
Comment (edwinjue) [reply components/Map/index.js:400]: Hi Anna

I want to let you know I updated the code block to remove `currentFactIndex` from the dependency array in the useEffect hook. Eslint added it back when i was saving changes but it have been empty. Thanks!

File: components/FactModal/index.jsx (hunk 1 above)
Comment (qnlee) [reply components/FactModal/index.jsx:31]: done!

File: components/Map/index.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (qnlee) [reply components/Map/index.js]: Not 100% sure that I understand what you mean in your 2nd comment, removed the `isLoading` prop for now though.

**Review by edwinjue — CHANGES_REQUESTED** [review]

**Review by edwinjue — APPROVED** [review]

**Pull request merged**